### PR TITLE
Add few recursive json

### PR DESCRIPTION
**Pull request by jingjingwang**
Merge after https://github.com/uwescience/myria/pull/776.

### CODE REVIEW

**Comment by jingjingwang** [comment]
@senderista Now this can be reviewed after #776! Basically I added a few recursive json queries. Once we have the system test framework, I will add tests for them.

File: jsonQueries/multiIDB_jwang/joinChain.json [hunk 1]
```diff
@@ -407,22 +395,8 @@
...
-                    "argChild": 44,
-                    "opId": 45,
-                    "opType": "CollectProducer"
-                }
-            ]
         }
     ],
     "logicalRa": "",
-    "rawQuery": ""
+    "rawQuery": "A(x, y) :- A0(x, y); A(x, y) :- R(x, z), A(z, y); B(x, y) :- B0(x, y), B(x, y) :- A(x, z), B(z, y); C(x, y) :- C0(x, y); C(x, y) :- B(x, z), C(z, y);"
```
Comment (senderista) [inline jsonQueries/multiIDB_jwang/joinChain.json]: Typo? Shouldn't

```
B(x, y) :- B0(x, y),
```

be

```
B(x, y) :- B0(x, y);
```

File: jsonQueries/multiIDB_jwang/joinCircle.json [hunk 2]
```diff
@@ -398,22 +391,8 @@
...
-                    "argChild": 45,
-                    "opId": 46,
-                    "opType": "CollectProducer"
-                }
-            ]
         }
     ],
     "logicalRa": "",
-    "rawQuery": ""
+    "rawQuery": "A(x, y) :- A0(x, y); A(x, y) :- C(x, z), A(z, y); B(x, y) :- B0(x, y), B(x, y) :- A(x, z), B(z, y); C(x, y) :- C0(x, y); C(x, y) :- B(x, z), C(z, y);"
```
Comment (senderista) [inline jsonQueries/multiIDB_jwang/joinCircle.json]: Typo? Shouldn't

```
B(x, y) :- B0(x, y),
```

be

```
B(x, y) :- B0(x, y);
```

**Comment by senderista** [comment]
@jingjingwang what happened to this? Was it already merged from another branch?

**Pull request merged**